.rubocop.yml

```diff
@@ -15,7 +22,7 @@ AllCops:
 Style/StringLiterals:
   EnforcedStyle: double_quotes
 
-Metrics/LineLength:
+Layout/LineLength:
```

Comment:
AFAIK, metrics are disabled completed in standard, so we can remove their configurations.